### ISSUE TITLE
Too many blank lines/empty spaces in web pages...

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#699]
**Reported by H.D. on 2010-06-13 16:47**
The NVDA should be able to suppress blank lines/spaces found in many web pages whenever such is possible. When navigating line by line or paragraph by paragraph, it is an extra work, awkward, or not so productive, to keep hitting down and/or up arrow to get to a web content if there are so many blank lines/spaces in between. Of course, the exception to that is in the "edit" field, for it is where one would want actual fill-in layout intagged.

**Comment by leonardder** [comment]
I suggest closing this in favour of #4904
